src/core/server/base.js

```diff
@@ -394,9 +397,21 @@ export function runServer({
         const routes = require(`${appName}/routes`).default;
         const createStore = require(`${appName}/store`).default;
         /* eslint-enable global-require, import/no-dynamic-require */
-        const server = baseServer(
+        let server = baseServer(
           routes, createStore, { appInstanceName: appName });
         if (listen === true) {
+          if (useHttpsForDev && (host === 'example.com')) {
+            const options = {
+              key: fs.readFileSync('bin/local-dev-server-certs/my-server.key.pem'),
+              cert: fs.readFileSync('bin/local-dev-server-certs/my-server.crt.pem'),
+              ca: fs.readFileSync('bin/local-dev-server-certs/my-private-root-ca.crt.pem'),
+              passphrase: '',
+            };
+            server = https.createServer(options, server);
+          } else {
+            log.info(
```

Comment:
This is in the wrong place. The statement up above should look more like:

```js
if (useHttpsForDev) {
  if (host === 'example.com') {
    // server = ...
  } else {
    log.info(`To use the HTTPS server you must...`);
  }
}
```

Reply:
Any reason for this? I feel having 2 logical statements that don't execute anything differently isn't needed.

Reply:
The log message is currently in the wrong place. For example, if `process.env.USE_HTTPS_FOR_DEV` was not set at all (or it was false) then you'd see a message in the console telling you that you are configuring HTTPS wrong. Logging a message in this case would be unnecessary and would be confusing.

Reply:
Wait, if we follow your suggestion, wouldn't that need an error message too?

Could we just do an ```else if``` and throw 2 different logs? If ```process.env.USE_HTTPS_FOR_DEV``` is ```False``` then we can just say ```https is not enabled```, then if the host is also not ```example.com``` then we can use the current log message.

Reply:
@kumar303 thoughts?